stat/roc.go

```diff
@@ -126,3 +128,62 @@ func ROC(cutoffs, y []float64, classes []bool, weights []float64) (tpr, fpr, thr
 
 	return tpr, fpr, cutoffs
 }
+
+// TOC returns the Total Operating Characteristic for the classes provided
+// and the minimum and maximum bounds for the TOC.
+//
```

Comment:
// If weights is nil, all weights are treated as 1.

Reply:
Done, and a warning about the panic (this needs to be added to ROC as well).

Reply:
And.... I've just discovered through a fumble that GitHub's branch protection doesn't work. The fix for ROC is now accidentally merged without a PR.